src/bci_build/package/__init__.py

```diff
@@ -402,12 +402,43 @@ def build_name(self) -> str | None:
 
     @property
     def build_version(self) -> str | None:
+        """Define the BuildVersion that is used for determining which container build
+        is newer than the other in the build service."""
+        # KIWI used to require an at least 3 component version X.Y.Z. For SLE, we set
+        # X.Y to MAJOR.MINOR and set Z to 0 for OsContainers. Derived
+        # containers inhert the base build_version X.Y and append a suffix .Z.ZZ
+
+        # It is important that the behavior for OsContainers is identical
+        # between KIWI and Dockerfile builds so that we can switch between these
+        # types.
+
+        if self.os_version.is_tumbleweed and isinstance(self, OsContainer):
+            return f"{str(datetime.datetime.now().year)}.0.0"
```

Comment:
Why just the year?

Reply:
I'd be fien with Y.Month, but that means a monthy "all tumbleweed containers get resubmitted" change and in the past Dan was very allergic to mass submissions

Reply:
Then I'd argue that the year is also useless and it should just be something like an actual version number that's bumped on $random_occasion or 84.87 (like filesystem).

Reply:
I don't get the 84.87 joke of filesystem. using datestamps for version numbers is very popular in various packages in tumbleweed. personally I don#t really care what we come up with but it needs to be `>= 2025.0`